docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -210,105 +205,97 @@ These items are:
 
 ### Smart contract class
 
-Now, review the smart contract in the `src/Square.ts` file. 
+Now, review the smart contract in the `src/Square.ts` file.
 
 The smart contract called `Square` has one element of on-chain state named `num` of type `Field` as defined by following code:
 
 ```ts src/Square.ts
-8
-9 export class Square extends SmartContract {
-10   @state(Field) num = State<Field>();
-11
-12 }
+export class Square extends SmartContract {
+  @state(Field) num = State<Field>();
+}
 ```
 
 zkApps can have up to eight fields of on-chain state. Each field stores up to 32 bytes (technically, 31.875 bytes or 255 bits) of arbitrary data. A later tutorial covers options for off-chain state.
 
 Now, this code adds the `init` method to set up the initial state of the smart contract on deployment:
 
 ```ts src/Square.ts
-8
-9 export class Square extends SmartContract {
-10   @state(Field) num = State<Field>();
-11
-12   init() {
-13     super.init();
-14     this.num.set(Field(3));
-15   }
-16
-17 }
+export class Square extends SmartContract {
```

Comment:
In the future, we'll be able to omit line numbers. For now, the Tutorial Runner requires line numbers for tutorial 1
Documented on our docs wiki https://github.com/o1-labs/docs2/wiki/Tutorial-Runner